driver/compile.go

```diff
@@ -543,7 +543,13 @@ func copyProcs(ps []ast.Proc) []ast.Proc {
 	return copies
 }
 
-func buildSplitFlowgraph(branch, tail []ast.Proc, mergeField field.Static, reverse bool, N int) *ast.SequentialProc {
+func buildSplitFlowgraph(branch, tail []ast.Proc, mergeField field.Static, reverse bool, N int) (*ast.SequentialProc, bool) {
+	if len(branch) == 0 {
+		return &ast.SequentialProc{
+			Node:  ast.Node{"SequentialProc"},
+			Procs: tail,
+		}, false
+	}
```

Comment:
That buildSplitFlowgraph now returns an unsplit flowgraph feels like a signal that we should refactor this, but let's save that for later.